integration/helpers/base_test.py

```diff
@@ -1,9 +1,12 @@
 import logging
 import os
 
+import requests
```

Comment:
CI is reporting a black formatting issue for this file

Reply:
will do a black reformatting.